### PR TITLE
NOX-693 Update StreetAddress to Include AddressLine3 Optional

### PR DESCRIPTION
**Pull request by msadikkoseaupa**

### CODE REVIEW

File: samples/Cryptocash.Api/DataSeed/Seeders/CryptocashCustomerDataSeeder.cs
Comment (rochar) [inline samples/Cryptocash.Api/DataSeed/Seeders/CryptocashCustomerDataSeeder.cs:1]: You need to recreate crypto cash migrations?

Comment (rochar) [reply samples/Cryptocash.Api/DataSeed/Seeders/CryptocashCustomerDataSeeder.cs:1]: Please  confirm that migrations and data seed are working properly

**Review by rochar — CHANGES_REQUESTED** [review]
looks good, two suggestions to review

**Review by gjorgji-grgovski-authoritypartners — DISMISSED** [review]

File: samples/Cryptocash.Api/DataSeed/Seeders/CryptocashCustomerDataSeeder.cs
Comment (msadikkoseaupa) [reply samples/Cryptocash.Api/DataSeed/Seeders/CryptocashCustomerDataSeeder.cs:1]: Migrations have been updated and confirmed

**Review by rochar — DISMISSED** [review]

**Review by gjorgji-grgovski-authoritypartners — DISMISSED** [review]

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]

**Review by rochar — APPROVED** [review]

**Pull request merged**